### DISCUSSION TITLE
Vertex Visibility computation

### DISCUSSION THREAD

**Discussion by trologat** [isl-org/Open3D#4488] [Q&A]
I would like to compute the vertex visibility of a triangle mesh from a certain camera. Any tips how I could achieve that in the fastest possible way? (python). Should I use the `RaycastingScene` or can I use a renderer for that?

**Comment by benjaminum** [comment]
You can use `RaycastingScene` for that. You can compute a depth map and then check if the distance of a vertex to the camera is smaller than the value in the depth map.
